src/transaction.ts

```diff
@@ -1,15 +1,4 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
-import { UserDatabaseName, UserDatabaseClient } from './user_database';
-import { WorkflowContextImpl } from './workflow';
-import { Span } from '@opentelemetry/sdk-trace-base';
-import { DBOSContext, DBOSContextImpl } from './context';
 import { ValuesOf } from './utils';
```

Comment:
We may want to consider moving stuff around once we get closer to releasing 3.0. In partucluar, we might want to move pure types into seperate files in order to facilitate using [`import type`](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export). You can use `import type` on individual imports, but it's easier to import type the entire file.